hazelcast/src/main/java/com/hazelcast/aggregation/impl/AbstractAggregator.java

```diff
@@ -73,7 +74,7 @@ public final void accumulate(I entry) {
                 }
                 accumulateExtracted(entry, results.get(i));
             }
-        } else {
+        } else if (!(extractedValue == NonTerminalJsonValue.INSTANCE)) {
```

Comment:
Extremely minor comment: this can be `extractedValue != NonTerminalJsonValue.INSTANCE`. I was a bit confused for a second :)

Reply:
good point